scripts/constants.sh

```
@@ -8,4 +8,8 @@ current_branch=${current_branch_temp////-}
 
 # camino-messenger-bot and caminoethvm git tag and sha
 git_commit=${CAMINO_BOT_COMMIT:-$(git rev-parse --short HEAD)}
-git_tag=${CAMINO_BOT_TAG:-$(git describe --tags --abbrev=0 --always || echo unknown)}
+git_tag=${CAMINO_BOT_TAG:-$(git describe --tags --always --dirty || echo unknown)}
+
+# get protocol releases from buf.build
+grpc_release=$(scripts/resolve_protocol_release.sh buf.build/gen/go/chain4travel/camino-messenger-protocol/grpc/go)
```

Shellcheck (0.10.0)

[warning] grpc_release appears unused. Verify use (or export if used externally).

(SC2034)